### PR TITLE
Use Faithlife.PortableContrib

### PR DESCRIPTION
**Pull request by ddunkin**

### CODE REVIEW

File: System.Data.SQLite.nuspec [hunk 1]
```diff
@@ -1,16 +1,16 @@
...
-    <owners>Logos Bible Software</owners>
+    <authors>Faithlife Corporation</authors>
+    <owners>Faithlife Corporation</owners>
     <requireLicenseAcceptance>false</requireLicenseAcceptance>
     <description>An independent implementation of the core of ADO.NET.</description>
     <copyright>Copyright 2014-2015 Faithlife / Logos Bible Software</copyright>
     <dependencies>
-      <group targetFramework="portable-net45+win+wp8+MonoAndroid+MonoTouch+Xamarin.iOS10">
-        <dependency id="Logos.PclContrib.Portable.Data" version="0.1.1" />
+      <group targetFramework="portable-net45+win+wp8">
```
Comment (ejball) [inline System.Data.SQLite.nuspec]: Should this be `portable-net45+netcore45+wpa81`?

File: System.Data.SQLite.nuspec [hunk 2]
```diff
@@ -20,7 +20,7 @@
     </dependencies>
   </metadata>
   <files>   
-    <file src="src\System.Data.SQLite-Portable\bin\$configuration$\System.Data.SQLite.*" target="lib\portable-net45+win+wp8+MonoAndroid+MonoTouch\" />
+    <file src="src\System.Data.SQLite-Portable\bin\$configuration$\System.Data.SQLite.*" target="lib\portable-net45+win+wp8\" />
```
Comment (ejball) [inline System.Data.SQLite.nuspec]: Should this be `portable-net45+netcore45+wpa81`?

File: src/System.Data.SQLite-Portable/System.Data.SQLite-Portable.csproj [hunk 3]
```diff
@@ -43,11 +43,13 @@
     <Compile Include="Thread.cs" />
   </ItemGroup>
   <ItemGroup>
```
Comment (ejball) [inline src/System.Data.SQLite-Portable/System.Data.SQLite-Portable.csproj]: Should we change to `Profile111` in this project?

Comment (ddunkin) [inline src/System.Data.SQLite-Portable/System.Data.SQLite-Portable.csproj]: Yes, now that PortableContrib is 111.

**Pull request merged**